README.md

```diff
@@ -49,10 +49,10 @@ For server-side applications or Node.js command line tools, include `'parse/node
 const Parse = require('parse/node');
 ```
 
-For React Native applications, include `'parse/react-native'`:
+For React Native applications, include `'parse/react-native.js'`:
 ```js
 // In a React Native application
-const Parse = require('parse/react-native');
+const Parse = require('parse/react-native.js');
```

Comment:
@dplewis before you ask, it is weird, but it does not work on react-native if imported as `require('parse/react-native')`. It does import `require('parse/reactnative')` if we change the name of the file to `reactnative.js`. I don't know exactly. Probably some bug in their side for resolving files with `-`.

Reply:
How did you figure this out?

Reply:
I used `npm link` to install the SDK in a test react native project that I built and I did some tests. In one of them, I added a `console.log` in the `react-native.js` file and it is only printing out when I import using `require('parse/react-native.js');`

Reply:
I make it a habit of adding file extensions in my projects. One of the shortcomings of node.